web/src/features/fba/components/infoPanel/AdvisoryReport.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Tabs, Tab, Grid } from '@mui/material'
+import { FireCenter } from 'api/fbaAPI'
+import { INFO_PANEL_CONTENT_BACKGROUND } from 'app/theme'
+import AdvisoryText from 'features/fba/components/infoPanel/AdvisoryText'
+import InfoAccordion from 'features/fba/components/infoPanel/InfoAccordion'
+import { DateTime } from 'luxon'
+import React, { useState } from 'react'
+
+interface AdvisoryReportProps {
+  issueDate: DateTime | null
+  forDate: DateTime
+  advisoryThreshold: number
+  selectedFireCenter?: FireCenter
+}
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+const TabPanel = ({ children, index, value }: TabPanelProps) => {
+  return (
+    <div hidden={value !== index} id={`tabpanel-${index}`}>
+      {value === index && <Box paddingBottom={3}>{children}</Box>}
+    </div>
+  )
+}
+
+const AdvisoryReport = ({ issueDate, forDate, advisoryThreshold, selectedFireCenter }: AdvisoryReportProps) => {
+  const [value, setValue] = useState(0)
```

Comment:
Minor: maybe name `value`, `setValue` to something like `tabNumber`, `setCurrentTabNumber` or something to clarify this state is for controlling tabs.